plugins/services/src/js/utils/CreateServiceModalFormUtil.js

```diff
@@ -34,6 +34,119 @@ const CreateServiceModalFormUtil = {
   },
 
   /**
+   * Patch an object with an object
+   *
+   * This function:
+   * - Removes items whose patch value is `empty` (null, undefined, 0, or '')
```

Comment:
@orlandohohmeier I will update the function, since currently `0` is considered empty.